src/version.txt

```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.0.1-rc.1"
```

Comment:
I don't think this change deserves a version bump. This does not, strictly speaking, impact the Packer configuration.

Reply:
I force-pushed to remove the version bump, deleted the tag for the release candidate, and deleted the prerelease.

And yes, we should do the same thing for skeleton-ansible-role.  I will create that PR now.

Reply:
See cisagov/skeleton-ansible-role#179.

Reply:
Can we resolve this conversation?